edge/pkg/edged/edged.go

```diff
@@ -50,9 +50,10 @@ import (
 	"k8s.io/client-go/util/workqueue"
 	internalapi "k8s.io/cri-api/pkg/apis"
 	"k8s.io/klog"
+	pluginwatcherapi "k8s.io/kubelet/pkg/apis/pluginregistration/v1"
```

Comment:
I don't know to comment on the entire PR or files, I assume all those changes are needed due to k8s version upgrade, right?

Reply:
Yes, we import the k8s api, so must adapt to its changes :)